src/sortByProps.js

```diff
@@ -0,0 +1,37 @@
+import { reduce, either, curry, prop, sort, comparator, map, lt } from 'ramda';
+
+import isNotArray from './isNotArray';
+
+/**
+ * Sort an list of objects by a list of props (if first prop value is equivalent, sort by second, etc).
+ *
+ * @func sortByProps
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.26.0|v2.26.0}
+ * @category List
+ * @sig [{k: v}] -> [String] -> [{k: v}]
+ * @param {[{k: v}]} list A list of objects to be sorted
+ * @param {[string]} props A list of properties in the list param to sort by
+ * @return {[{k: v}]} A new list sorted by the properties in the props param
+ * @example
+ *
+ * sortByProps(['num'], [{num: 3}, {num: 2}, {num: 1}])
+ * //=> [{num: 1}, {num: 2} {num: 3}]
+ * sortByProps(['letter', 'num'], [{num: 3, letter: 'a'}, {num: 2, letter: 'a'} {num: 1, letter: 'z'}])
+ * //=> [ {num: 2, letter: 'a'}, {num: 3, letter: 'a'}, {num: 1, letter: 'z'}]
+ * sortByProps(['name', 'num'], [{num: 3}, {num: 2}, {num: 1}])
+ * //=> [{num: 1}, {num: 2}, {num: 3}]
+ */
+
+const sortByProps = curry((list, props) => {
```

Comment:
You need to switch the functions arguments. In Functional programming data is always the last argument
```suggestion
const sortByProps = curry((props, list) => {
```

Reply:
Your examples are fine though